pages/instructorAssessmentDownloads/instructorAssessmentDownloads.sql

```diff
@@ -168,8 +165,7 @@ WITH all_submissions AS (
         JOIN assessment_instances AS ai ON (ai.assessment_id = a.id)
         LEFT JOIN group_configs AS gc ON (gc.assessment_id = a.id)
         LEFT JOIN groups AS g ON (g.id = ai.group_id AND g.group_config_id = gc.id)
-        LEFT JOIN group_users AS gu ON (gu.group_id = g.id)
-        JOIN users AS u ON (u.user_id = ai.user_id OR u.user_id = gu.user_id)
+        LEFT JOIN users AS u ON (u.user_id = ai.user_id)
```

Comment:
Should we also include a `LEFT JOIN users AS authn_u ON (u.user_id = s.auth_user_id)` and select `authn_u.uid AS submitter_uid`? If we include `submitter_uid` as a column in the output CSV then it will show the actual submitting user for both group work and also non group work (e.g., maybe a TA makes a submission for a student).

This is not a required change for this PR -- it's just a thought I had while reading the PR.

Reply:
Isn't this what `submission_user` is about? Not my change, was there already...

Reply:
D'oh! 😄